### PR TITLE
Language rework reworked

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -194,102 +194,102 @@
...
+    <string name="dont_show_boost">Don\'t show boosts</string>
+    <string name="dont_show_reply">Don\'t show replies</string>
     <string name="regex_filter">Regex filter</string>
     <string name="regex_error">Regex error</string>
     <string name="notification_delete">Empty notifications</string>
-    <string name="confirm_delete_notification">This will empty your notification on your instance.\nAre you sure?</string>
+    <string name="confirm_delete_notification">Empty all notifications on instance?</string>
     <string name="search_for">Search %1$s</string>
-    <string name="spoil_visibility_for_account">Visibility is spoiled for account.</string>
+    <string name="spoil_visibility_for_account">The account is visible.</string>
```
Comment (tateisu) [inline app/src/main/res/values/strings.xml]: no, this message is used when switching account in post composing activity. if switched account has more smaller visibility, visibility selection is changed to  spoiled.

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -194,102 +194,102 @@
...
-    <string name="confirm_block_user">User \"%1$s\" will be blocked. Are you sure?</string>
-    <string name="confirm_mute_user">User \"%1$s\" will be muted. Are you sure?</string>
-    <string name="confirm_delete_status">This toot will be deleted. Are you sure?</string>
-    <string name="nickname_label">Nickname and color (if specified, it\'s shown instead of full acct)</string>
-    <string name="color">Color</string>
+    <string name="confirm_block_user">Block %1$s?</string>
+    <string name="confirm_mute_user">Mute %1$s?</string>
+    <string name="confirm_delete_status">Delete toot?</string>
+    <string name="nickname_label">Nickname and colour (if specified, it\'s shown instead of full acct)</string>
+    <string name="color">Colour</string>
```
Comment (tateisu) [inline app/src/main/res/values/strings.xml]: changing from "color" to "colour" is not accepted. 

- "favourite" word is used in Mastodon official.
- "colour" word is NOT used in Mastodon official.

In this case I should obey to Mastodon wording.

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -302,102 +302,103 @@
...
     <string name="pick_image">Pick an image</string>
     <string name="pick_images">Pick image(s)…</string>
 
     <string name="image_alpha">Image alpha</string>
     <string name="image">Image</string>
     <string name="column_header">Column header</string>
-    <string name="foreground_color">Foreground color</string>
-    <string name="hashtag_and_visibility_not_match">this message contains hashtags, but message visibility is not public. normally hashtags are searchable only in public messages. Are you sure?</string>
+    <string name="foreground_color">Foreground colour</string>
+    <string name="hashtag_and_visibility_not_match">Send non-public toot containing hashtags, (which are not normally searchable in public messages)?</string>
```
Comment (tateisu) [inline app/src/main/res/values/strings.xml]: This sentence will be a problem when translating to other languages.

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -302,102 +302,103 @@
...
     <string name="copy">Copy</string>
     <string name="send">Send</string>
     <string name="mute_word">Ban word</string>
     <string name="select_and_copy">Select and copy</string>
     <string name="search_web">Web search</string>
     <string name="muted_word">Banned words</string>
-    <string name="word_was_muted">The word was banned.</string>
+    <string name="word_was_muted">Banned.</string>
```
Comment (tateisu) [inline app/src/main/res/values/strings.xml]: too short.

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -302,102 +302,103 @@
...
-    <string name="disable_tablet_mode">Disable tablet mode (app restart required)</string>
-    <string name="minimum_column_width">Minimum column width (default=300(dp),app restart required)</string>
-    <string name="media_thumbnail_height">Media Thumbnail height (default=64(dp), app restart required)</string>
-    <string name="account_picker_reply">Which account do you reply from?</string>
+    <string name="cant_close_column_by_back_button_when_multiple_column_shown">You can\'t close the column using the \"Back\" button with multiple columns visible.</string>
+    <string name="disable_tablet_mode">Turn off tablet mode (app restart required)</string>
+    <!-- Density-independent pixels, written as dp (pronounced “dips”), are flexible units that scale to have uniform dimensions on any screen. https://material.io/design/layout/understanding-layout.html#pixel-density -->
+    <string name="minimum_column_width">Minimum column width (default=300 dp), (app restart required)</string>
+    <string name="media_thumbnail_height">Minimum column width (default=300 dp), (app restart required)</string>
```
Comment (tateisu) [inline app/src/main/res/values/strings.xml]: missed.

File: app/src/main/res/values/strings.xml [hunk 6]
```diff
@@ -302,102 +302,103 @@
...
+    <string name="attachment_in_draft_is_lost">Some of the media attachments in the draft have been lost.</string>
     <string name="app_data_export">Export app data</string>
     <string name="app_data_import">Import app data</string>
-    <string name="app_data_import_desc">You can export data to any app that can receive data, But maybe you can import data only from device\'s storage or Google drive.\nColumn background image may be not exported/imported over device.\nTimeline font is not exported/imported.\nCustom notification sound may be not exported/imported over device.</string>
-    <string name="app_data_import_confirm">existing data ( before import ) will be cleared. Are you sure?</string>
-    <string name="user_id_conversion_failed">User id conversion failed.</string>
+    <string name="app_data_import_desc">You can export data to any app that can receive data, but maybe you can import data only from device\'s storage or a cloud service.\nColumn background image may be not exported/imported over device.\nTimeline font is not exported/imported.\nCustom notification sound may be not exported/imported over device.</string>
```
Comment (tateisu) [inline app/src/main/res/values/strings.xml]: No, we can't use any cloud service. I tried some cloud service and only GD can be used.

**Comment by tateisu** [comment]
Your PR can not be merged at once because changes are so several.
I will modify the wording little by little referring to your pointed out.

File: app/src/main/res/values/strings.xml [hunk 7]
```diff
@@ -553,65 +554,65 @@
...
-    <string name="cant_add_list_follow_requesting">Can\'t add user to list:\nPlease retry 2…3 times. Or if user is \"locked\", please wait until follow request is approved.</string>
-    <string name="cant_add_list_follow_failed">Can\'t add user to list:\nfollow operation failed.</string>
-    <string name="list_member_added">User has been added to the list.</string>
-    <string name="confirm_mute_notification_for_user">Mute notification from this user</string>
-    <string name="account_moved_to">%1$s has moved to:</string>
-    <string name="jump_moved_user">%1$s has been moved. Would you like to check %2$s ?</string>
+    <string name="account_sync_failed">Could not sync account.</string>
+    <string name="list_retry_with_follow">Try to follow and add %1$s to list?</string>
+    <string name="cant_add_list_follow_requesting">Can\'t add user to list:\nPlease retry 2,3 times. Or if user is \"Locked\", please wait until for approval of follow request.</string>
+    <string name="cant_add_list_follow_failed">Can\'t add user to list:\nRequest declined.</string>
```
Comment (tateisu) [inline app/src/main/res/values/strings.xml]: no, there are some reason for follow request failed.

File: app/src/main/res/values/strings.xml [hunk 8]
```diff
@@ -553,65 +554,65 @@
...
     <string name="list_not_created">(list is not created)</string>
     <string name="your_lists">Your lists</string>
-    <string name="media_attachment_max_byte_size">Media attachment (image) maximum bytes size (unit:Mega bytes. default is 8)</string>
-    <string name="media_attachment_max_byte_size_movie">Media attachment (movie) maximum bytes size (unit:Mega bytes. default is 40)</string>
-    <string name="tootsearch">tootsearch(JP)</string>
-    <string name="cant_handle_uri_of">Subway Tooter can\'t handle URI \"%1$s\". Please select other app.</string>
+    <string name="media_attachment_max_byte_size">Media attachment (image) size limit. Unit: MB. The default is 8)</string>
+    <string name="media_attachment_max_byte_size_movie">Media attachment (movie) size limit. Unit: MB. The default is 40)</string>
+    <string name="tootsearch">tootsearch (JP)</string>
+    <string name="cant_handle_uri_of">Cannot open the \"%1$s\" media attachment. Try to open it in a web browser.</string>
```
Comment (tateisu) [inline app/src/main/res/values/strings.xml]: this message is not used to media attachment.

File: app/src/main/res/values/strings.xml [hunk 9]
```diff
@@ -623,25 +624,25 @@
     <string name="notification_type_reaction">reaction</string>
     <string name="notification_type_vote">vote</string>
     <string name="dont_show_normal_toot">Don\'t show normal toot</string>
-    <string name="set_focus_point">set focus point (Mastodon 2.3 or later)</string>
-    <string name="report_forward_desc">The account is from another instance. Send an anonymized copy of the report there as well?</string>
+    <string name="set_focus_point">set focus point (Mastodon 2.3+)</string>
+    <string name="report_forward_desc">The account is from another instance. Send an anonymised copy of the report there as well?</string>
```
Comment (tateisu) [inline app/src/main/res/values/strings.xml]: use "anonymized" to obey Mastodon official.

File: app/src/main/res/values/strings.xml [hunk 10]
```diff
@@ -705,9 +706,9 @@
 
     <string name="filter_options">Options</string>
     <string name="filter_irreversible">Irreversible</string>
-    <string name="filter_irreversible_long">Irreversible. Drop instead of hide. Filtered toots will disappear irreversibly, even if filter is later removed.</string>
-    <string name="filter_word_match">Whole word</string>
-    <string name="filter_word_match_long">Whole word. Recommended to enable only phrases consisting of alphanumeric characters.</string>
+    <string name="filter_irreversible_long">Irreversible. Drop instead of hide. Filtered toots are lost forever, even if the filter is removed later.</string>
+    <string name="filter_word_match">The whole word</string>
```
Comment (tateisu) [inline app/src/main/res/values/strings.xml]: obey to Mastodon official.

**Comment by tateisu** [comment]
some strings are cherry-picked at https://github.com/tateisu/SubwayTooter/commit/740258049b811148bba4c9bfbd6cab99cb7840b7 .

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (comradekingu) [reply app/src/main/res/values/strings.xml]: I don't understand this. Could possibly be "spoilt".
What happens exactly?

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (comradekingu) [reply app/src/main/res/values/strings.xml]: Both "favourite" and "colour" are British English. It makes sense to stick those or
"favorite" and "color".

File: app/src/main/res/values/strings.xml (hunk 3 above)
Comment (comradekingu) [reply app/src/main/res/values/strings.xml]: How so?

File: app/src/main/res/values/strings.xml (hunk 4 above)
Comment (comradekingu) [reply app/src/main/res/values/strings.xml]: Why?

File: app/src/main/res/values/strings.xml (hunk 6 above)
Comment (comradekingu) [reply app/src/main/res/values/strings.xml]: So the support is somehow specific to Google Drive?

File: app/src/main/res/values/strings.xml (hunk 10 above)
Comment (comradekingu) [reply app/src/main/res/values/strings.xml]: That is a marked noun error if you ask me. What seems to be the issue?

File: app/src/main/res/values/strings.xml [hunk 11]
```diff
@@ -826,8 +828,7 @@
     <string name="about_this_instance">About this instance</string>
     <string name="top_page">Top page</string>
     <string name="show_instance_ticker">Show #InstanceTicker (column reload required)</string>
-    <string name="instance_ticker_copyright">#InstanceTicker\nhttps://cdn.weep.me/mastodon/\nCopyright 2018 weepjp, kyori19.</string>
+    <string name="instance_ticker_copyright">#InstanceTicker\nhttps://cdn.weep.me/mastodon/\nCopyright © 2018 weep.jp and kyori19.</string>
```
[the remark this answers is not in the document]
Comment (comradekingu) [reply app/src/main/res/values/strings.xml]: That isn't how the nickname is spelled elsewhere though…

File: app/src/main/res/values/strings.xml (hunk 5 above)
Comment (comradekingu) [reply app/src/main/res/values/strings.xml]: I don't understand what I (?) missed?

File: app/src/main/res/values/strings.xml (hunk 7 above)
Comment (comradekingu) [reply app/src/main/res/values/strings.xml]: No matter the reason, the end result is a declined request.

**Pull request closed without merging**